lib/ecstatic/defaults.json

```diff
@@ -3,7 +3,7 @@
   "showDir": true,
   "showDotfiles": true,
   "humanReadable": true,
-  "hidePermissions": false,
+  "selectDisplay":"icon|permissions|filesize|filename",
```

Comment:
I think this API should take an object:

```js
{ icon: true, permissions: true, filesize: true, filename: true }
```

something like this. On the CLI, I could be convinced to take on this API, though we have an existing mime types API that takes JSON on the CLI, so that's worth considering...